main.roc

```diff
@@ -0,0 +1,80 @@
+app "generate-build"
+    packages { pf: "/home/username/gitrepos/roc/examples/static-site-gen/platform/main.roc" }
+    imports [
+        pf.Html.{ html,svg, main, p, footer, h1,  head, header, body, div, text, a, link, meta, title },
+        pf.Html.Attributes.{ role, attribute, name, content, href, rel, lang, class,  charset },
+    ]
+    provides [transformFileContent] to pf
+
+transformFileContent : Str, Str -> Str
+transformFileContent = \fileName, htmlContent -> Html.render (view fileName htmlContent)
+
+view : Str, Str -> Html.Node
+view = \fileName, htmlContent ->
+    viewContent = 
+        when fileName is 
+            "index.html" -> viewIndex htmlContent
+            _ -> viewExample htmlContent
+    
+    html [lang "en"] [
+        head [] [
+            title [] [text "title"],
+            meta [charset "utf-8"] [],
+            meta [name "viewport", content "width=device-width"] [],
+            link [rel "icon", href "/favicon.svg"] [],
+            link [rel "stylesheet", href "/styles.css"] [],
+        ],
+        body [] [
+            div [class "top-header-extension"] [],
+            header [class "top-header"] [
+                div [class "pkg-and-logo"] [
+                    a [class "logo", href "#"] [logoSvg],
+                    h1 [class "pkg-full-name"] [
+                        a [href "#"] [text "Examples"],
```

Comment:
I should have mentioned, the logo and Examples page heading links do nothing, we will need to update this when we integrate into site.